javagen/src/main/java/com/azure/autorest/model/clientmodel/Package.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.model.clientmodel;
+
+import com.azure.autorest.extension.base.plugin.JavaSettings;
+
+public class Package {
+
+    public static final String GENERIC_CORE_PACKAGE_NAME = "com.generic.core";
+    public static final String GENERIC_JSON_PACKAGE_NAME = "com.generic.json";
+
+    public static final String AZURE_CORE_PACKAGE_NAME = "com.azure.core";
+    public static final String AZURE_JSON_PACKAGE_NAME = "com.azure.json";
+
+
+    public static final Package CORE = new Builder()
+            .packageName(GENERIC_CORE_PACKAGE_NAME)
+            .groupId("com.generic")
+            .artifactId("generic-core")
+            .build();
+
+    public static final Package JSON = new Builder()
+            .packageName(GENERIC_JSON_PACKAGE_NAME)
+            .groupId("com.generic")
+            .artifactId("generic-json")
+            .build();
```

Comment:
Currently, we use this for module-info and pom.xml

The value will be changed to azure-core lib if branding=true